specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json

```diff
@@ -19,7 +19,7 @@
     "201": {
       "body": {
         "name": "AzurePrivatePeering",
-        "id": "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/CrossConnection-SiliconValley/providers/Microsoft.Network/expressRouteCrossConnections/CrossConnection-6e918437-5077-4ffe-b45f-0b7c0672da4b/peerings/AzurePrivatePeering",
+        "id": "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/CrossConnection-SiliconValley/providers/Microsoft.Network/expressRouteCrossConnections/6e918437-5077-4ffe-b45f-0b7c0672da4b/peerings/AzurePrivatePeering",
```

Comment:
Same comment as above for the `subscriptionId` here

Reply:
Fixed